docs/spatial-audio-rfc.md

```diff
@@ -59,10 +61,12 @@ aligned(8) class SpatialAudioBox extends Box(‘SA3D’) {
 
 - `channel_map` is a sequence of 32-bit unsigned integers that maps audio channels in a given audio track to ambisonic components, given the defined `ambisonic_channel_ordering`. The sequence of `channel_map` values should match the channel sequence within the given audio track.
 
-  For example, consider a 4-channel audio track containing ambisonic components *W*, *X*, *Y*, *Z* at channel indexes 0, 1, 2, 3, respectively. For `ambisonic_channel_ordering = 0` (ACN), the ordering of components should be *W*, *Y*, *Z*, *X*, so the `channel_map` sequence should be `0`, `2`, `3`, `1`.
+  For the example case of ambisonic_type == 'Periphonic', consider a 4-channel audio track containing ambisonic components *W*, *X*, *Y*, *Z* at channel indexes 0, 1, 2, 3, respectively. For `ambisonic_channel_ordering = 0` (ACN), the ordering of components should be *W*, *Y*, *Z*, *X*, so the `channel_map` sequence should be `0`, `2`, `3`, `1`.
```

Comment:
Format nit: `ambisonic_type = 0` (Periphonic)

Also I may consider being more explicit where we have `non_diegetic = 0` as it took me a second scan to spot the difference.